src/app/components/story-edition-date-label/story-edition-date-label.component.ts

```diff
@@ -27,6 +27,6 @@ import { CommonModule, NgIf } from '@angular/common';
 	imports: [CommonModule, NgIf],
 })
 export class StoryEditionDateLabelComponent {
-	@Input() label: string | undefined;
-	@Input() markAsNew: boolean = false;
+	label = input<string | undefined>();
```

Comment:
When migrating to signals we do not need to specify `undefined` as a possible type for the inputs when they're not required. Please keep this one as `label = input<string>` and we're good to go.

Reply:
Done!!